ajax/libs/gzip-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gzip-js",
+  "description": "GZIP in pure JavaScript (works in the browser)",
+  "filename": "gzip.min.js",
+  "keywords": [
+    "gzip",
+    "zip",
+    "pure javascript"
+  ],
+  "author": "T. Jameson Little <t.jameson.little@gmail.com>",
+  "homepage": "https://manifestwebdesign.github.io/angular-gridster",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/beatgammit/gzip-js.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/beatgammit/gzip-js.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "*.js"
+        ]
+      }
+    ]
+  },
+  "license": "GPL"
```

Comment:
Please modify this to 
```
  "licenses": [
    "MIT",
    "GPL-2.0"
  ]
```
according to https://github.com/beatgammit/gzip-js/blob/master/LICENSE#L3 and https://github.com/beatgammit/gzip-js/blob/master/LICENSE.MIT

Reply:
@extend1994 I used this lib for zip array and store it in session storage, and as I remember it worked well. However, after review the author comments I doubt if the library solution is well formed, also the library has no maintenance for 6 years, thus I suggest not to add it for now. If in the future any one else will ask for it we could continue from this point.
@extend1994 @PeterDaveHello For your discretion guys!

Reply:
@sashberd I think it's fine, many people still use this lib as https://www.npmjs.com/package/gzip-js page show.
Licenses part revise as @extend1994's suggestion is good.
What do you think?

Thank you.

Reply:
@sashberd I will try to open an issue to confirm this. Maybe wait it for a few days then make a decision.
@sufuf3 I think @sashberd was talking about should we add this library for now. Because it seem incompatible with browser. Not argue license part.